sonoff/user_config.h

```diff
@@ -267,6 +267,7 @@
   #define USE_BMP                                // Add I2C code for BMP085/BMP180/BMP280/BME280 sensor (+4k code)
 //    #define USE_BME680                           // Add additional support for BME680 sensor using Bosch BME680 library (+4k code)
   #define USE_SGP30                              // Add I2C code for SGP30 sensor (+1k1 code)
+  #define USE_VL53L1X                              // Add I2C code for VL53L1X sensor
```

Comment:
@arendst how do I get the compiled size code difference?

Reply:
Compile once without the define and once with the define and substract the compiled file sizes.